bower.json

```diff
@@ -25,21 +25,30 @@
     "tests"
   ],
   "dependencies": {
-    "angular-datatables": "~0.4.3",
-    "angular": "~1.3.15",
-    "datatables": "~1.10.7",
-    "jquery": "~2.1.4",
-    "angular-highlightjs": "~0.4.1",
-    "highlightjs": "~8.5.0",
-    "angular-loading-spinner": "~0.0.1",
-    "angular-route": "~1.3.15",
-    "spin.js": "",
+    "angular-datatables": "~0.5.4",
+    "angular": "~1.5.3",
+    "datatables": "~1.10.11",
+    "jquery": "~2.2.3",
+    "angular-highlightjs": "~0.6.1",
+    "highlightjs": "~9.2.0",
+    "angular-loading-spinner": "~0.0.3",
+    "angular-route": "~1.5.3",
+    "spin.js": "~2.3.2",
     "angular-ui-ace": "~0.2.3",
-    "ace-builds": "~1.2.0",
-    "bootstrap": "~3.3.4",
+    "ace-builds": "~1.2.3",
+    "bootstrap": "~3.3.6",
     "jquery-ui": "~1.11.4",
     "jstree-directive": "~0.1.5",
-    "ng-context-menu": "~1.0.1",
+    "ng-context-menu": "~1.0.3",
     "underscore": "~1.8.3"
+  },
+  "resolutions": {
+    "angular-datatables": "~0.5.4",
+    "angular": "~1.5.3",
+    "jquery": "~2.2.3",
+    "angular-highlightjs": "~0.6.1",
+    "highlightjs": "~9.2.0",
+    "angular-route": "~1.5.3",
+    "spin.js": "~2.3.2"
   }
 }
```

Comment:
I have a question about this commit.
These updates are based on what? You just pick the latest release of each module and test it?